### PR TITLE
Update signNow

### PR DESCRIPTION
**Pull request by 40504**

### CODE REVIEW

**Review by ApCoder123 — COMMENTED** [review]
The doc page describes how to protect a specific document with "_dual factor authentication_". This seems to force the signer to receive a text to the number that the sender has specified before they can sign the document. I don't believe that this meets our definitions of 2FA, as it does not affect the user's sign in process & is not protecting the account with a redundant layer of authentication. Please confirm whether the account has 2FA or just specific documents.

**Comment by 40504** [comment]
Just created an account and can confirm:

File: _data/other.yml [hunk 1]
```diff
@@ -606,11 +606,12 @@ websites:
...
-  - name: SignNow
+  - name: signNow
     url: https://www.signnow.com/
-    img: signnow.png
-    twitter: signnow
-    facebook: esignaturesignnow
+    img: signnow.svg
+    tfa:
+      - email
+    doc: https://www.signnow.com/features/setup-dual-factor-authentication
```
Comment (ApCoder123) [inline _data/other.yml]:
```suggestion

```
I think we can remove the doc link - it doesn't have any useful info about the 2FA features that you showed in the screenshot.

**Review by ApCoder123 — CHANGES_REQUESTED** [review]

**Comment by 40504** [comment]
done

**Review by ApCoder123 — APPROVED** [review]

**Review by phallobst — APPROVED** [review]

**Pull request merged**